/camera_intrin_info_node:
    ros__parameters:
      zoom00: '{"zoom": "00", "camera_matrix": [608.63231, 0.0, 298.48801, 0.0, 813.4077, 262.04761, 0.0, 0.0, 1.0]}'
      zoom01: '{"zoom": "01", "camera_matrix": [693.77809, 0.0, 293.51689, 0.0, 926.50309, 237.6301, 0.0, 0.0, 1.0]}'
      zoom02: '{"zoom": "02", "camera_matrix": [838.12836, 0.0, 307.6042, 0.0, 1116.95159, 252.04598, 0.0, 0.0, 1.0]}'
      zoom03: '{"zoom": "03", "camera_matrix": [1096.71501, 0.0, 315.56908, 0.0, 1461.86779, 177.50954, 0.0, 0.0, 1.0]}'
      zoom04: '{"zoom": "04", "camera_matrix": [1226.37397, 0.0, 296.52369, 0.0, 1652.30319, 353.87703, 0.0, 0.0, 1.0]}'
      zoom05: '{"zoom": "05", "camera_matrix": [1792.5891, 0.0, 288.99308, 0.0, 2376.28713, 111.31859, 0.0, 0.0, 1.]}'
      zoom06: '{"zoom": "06", "camera_matrix": [2474.25488, 0.0, 23.58464, 0.0, 3294.20274, 252.06849, 0.0, 0.0, 1.0]}'
      zoom07: '{"zoom": "07", "camera_matrix": [3490.02409, 0.0, 274.67275, 0.0, 4650.30229, 280.45259, 0.0, 0.0, 1.0]}'
      zoom08: '{"zoom": "08", "camera_matrix": [5094.58546, 0.0, 420.48448, 0.0, 6767.01277, 233.67731, 0.0, 0.0, 1.0]}'
      zoom09: '{"zoom": "09", "camera_matrix": [8690.17938, 0.0, 109.43162, 0.0, 11482.58181, 95.96445, 0.0, 0.0, 1.0]}'
      zoom10: '{"zoom": "10", "camera_matrix": [11547.24362, 0.0, 290.38308, 0.0, 15344.18277, 59.84425, 0.0, 0.0, 1.0]}'
      zoom11: '{"zoom": "11", "camera_matrix": [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]}'
      zoom_fixed: '{"zoom": "fixed", "camera_matrix": [897.711430, 0.000000, 330.535776, 0.000000, 895.153341, 233.408292, 0.000000, 0.000000, 1.000000]}'
      binocular_left: '{"zoom": "binocular_left", "camera_matrix": [451.66294, 0.0, 336.83788, 0.0, 451.58789, 214.38401, 0.0, 0.0, 1.0]}'
      gazebo_left: '{"zoom": "gazebo_left", "camera_matrix": [876.64389, 0.0, 320.5, 0.0, 876.64389, 240.5, 0.0, 0.0, 1.0]}'
